back-end/src/routes/api.js

```diff
@@ -64,32 +64,25 @@ const uploadFile = (request, response) => {
             labelNames.push(label.Name);
     
             // Set up dynamoDB params
-            var tagParams = {
-              TableName:"Tags",
+            const d = new Date();
+            var uploadDate = (d.getMonth() + 1) + "/" + d.getDate() + "/" + d.getFullYear();
+            var fileParams = {
+              TableName:"Files",
               Item:{
+                "file": newFileName,
                 "tag": label.Name,
-                "file": newFileName
+                "email": request.body.email,
+                "title": request.body.title,
+                "desc": request.body.desc,
+                "likes": 0,
+                "uploadDate": uploadDate
               }
             };
     
             // Put tag item in table so we can query for files based on tags.
-            const tagData = await dynamodb.put(tagParams).promise();
-            console.debug("Added Item: ", tagData);
+            const fileData = await dynamodb.put(fileParams).promise();
```

Comment:
do you need the `.promise()`? referencing [here](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/dynamodb-example-document-client.html#dynamodb-example-document-client-put)

Reply:
.promise() returns promises so I can avoid using callbacks.